code/src/index.css

```diff
@@ -11,3 +19,18 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+canvas {
+  height: 0;
+}
+
+video {
+  width: 100%;
+  margin: auto;
+}
```

Comment:
you can nest these two parts into the styled div called Wrap in BarcodeScanner.js